CMakeLists.txt

```diff
@@ -28,6 +26,12 @@ option (SPEC_ARPA2
 	"Build inlucde files from ASN.1 specs for ARPA2 projects"
 	ON)
 
+set(Quick-DER_MAJOR_VERSION 0)
+set(Quick-DER_MINOR_VERSION 1)
+set(Quick-DER_PATCH_VERSION 5)  # RC5, will move to 0.2 for release
+set(Quick-DER_VERSION
+	${Quick-DER_MAJOR_VERSION}.${Quick-DER_MINOR_VERSION}.${Quick-DER_PATCH_VERSION})
+
```

Comment:
De patchnummers, hebben we gesteld, mogen door packagers worden overruled door hun eigen packaging serienummerreeks, maar overnemen van de `RCn`, `alphaN` of `betaN` is natuurlijk een aardige default.  Zonodig met een Git tag toegevoegd.
Let op dat dit &mdash;heel eigenwijs&mdash; het patch nummer op `local-YYYYMMDD-hhmmss` vastpint als je Git niet gecommit is.  Dat lijkt me typisch iets dat we een tijdje proberen en dan, al dan niet horendol gedraaid, torpederen, negeren, accepteren, tolereren, appreciëren of zelfs declameren.